test/test_autograd.py

```diff
@@ -2150,6 +2150,10 @@ def run_test(input_size, exponent):
         run_test((10, 10), torch.zeros(10, 10))
         run_test((10,), 0)
 
+    def test_pow_scalar_base(self):
+        a = torch.tensor([3.], requires_grad=True)
```

Comment:
Actually can you please make this a larger random tensor (think 10 elements)? Just to be sure.